### ISSUE TITLE
Multi fields

### ISSUE THREAD

**Issue by loveorigami** [bupy7/yii2-dynamic-fields#1]
Thanks for usefull widget.
For me need add more then one fields to the widget.(as here - https://github.com/unclead/yii2-multiple-input/blob/master/docs/multiple_input_multiple.md)
For exapmple:
- Model -> video
- youtube_id | title

May be it will be make as

``` php
echo DynaFields::widget([
    'urlAdd' => ['your-action-add'],
    'urlRemove' => ['your-action-remove'],
    'inputMethod' => 'textInput',
    'inputMethodArgs' => [['maxlength' => true]],
    'form' => $form,
    'models' => $models,
    'attribute' => ['youtube_id', 'title'],
]);
```

What do you think?

**Comment by bupy7** [comment]
@loveorigami widget designed only for one field.

**Comment by loveorigami** [comment]
Yes, I understand that. I offered to do for many fields. Because saving models with many fields often (in my project).

In the previous widget because the field stored in the same model.

Our widgets are stored in the input fields of another model.

**Comment by bupy7** [comment]
It now my is not planned. But you can make fork and offered pull-request with this solution. If it solution will be good, then I do make merge. =)

**Comment by bupy7** [comment]
In widget must be all replaced. It will be new major version.
